### DISCUSSION TITLE
fzf-lua can't get icons colors

### DISCUSSION THREAD

**Discussion by GreenSleeper** [NvChad/NvChad#2155] [Q&A]
Any one use [`fzf-lua`](https://github.com/ibhagwan/fzf-lua) with `NvChad`?
Normally, `fzf-lua` can't get colors of icons automaticlly, but with `NvChad`, it can't.

**Comment by siduck** [comment]
@GreenSleeper  can  you try to disable lazyloading of nvim-web-devicons and see? ( lazy=false)

**Comment by GreenSleeper** [comment]
> can you try to disable lazyloading of nvim-web-devicons and see? ( lazy=false)

not working.

**Reply by siduck** [reply]
i cant think of other reasons :/
